website/docs/reference/widgets/video.md

```diff
@@ -18,7 +18,14 @@ These properties are customizable options present in the property pane of the wi
 
 <dd>
 
-Allows you to set the video source to be played. 
+Allows you to set the video source to be played. Supports the following URL components:
+
+* Protocol
+* Auth
+* Hostname or IP
+* Port
+* Path/params
+
 
```

Comment:
Let's simplify to say that it now supports IP based URL. Let's also tag examples for the same